reflex/event.py

```diff
@@ -1058,8 +1058,8 @@ def get_hydrate_event(state) -> str:
 
 
 def call_event_handler(
-    event_handler: EventHandler | EventSpec,
-    arg_spec: ArgsSpec | Sequence[ArgsSpec],
+    event_callback: EventHandler | EventSpec,
+    event_spec: ArgsSpec | Sequence[ArgsSpec],
```

Comment:
Won't it be confusing to have 
`event_callback` of type `EventSpec`
but then
`event_spec` of type `ArgsSpec` ?

Tho I'm not sure what we could use instead, so maybe it's fine.

Reply:
I would love to change existing terminology but that's the best i could do while working in the system :/